### DISCUSSION TITLE
Payment receipts per year are expressed in US dollars, not local currency

### DISCUSSION THREAD

**Discussion by JohnTitor** [community/community#47937] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The feature to output the payment receipts by year is very useful and helpful, thank you!

However, the amounts are in U.S. dollars, while the receipt by month uses my local currency, Japanese yen.
That's kinda inconvenient to calculate my taxes, is it possible to show the amount in local currency just like monthly receipts?

**Comment by Zimbabar** [comment]
Payment receipts per year are an important aspect of financial record keeping for any business. However, if these receipts are not expressed in US dollars, it can cause confusion and difficulties when dealing with financial transactions. To solve this issue, businesses should consider locating a [nearby Chase Bank](https://mynearestbank.com/chase-bank-near-me/). Chase Bank offers foreign currency exchange services to its customers, making it easier for businesses to convert their payment receipts into US dollars. With Chase Bank's help, businesses can have a better understanding of their financial transactions and maintain more accurate financial records. Therefore, it is highly recommended that businesses should use the services of Chase Bank to convert their payment receipts into US dollars. This will help them keep their financial records up-to-date and avoid any potential complications in the future.You can Locate your nearest bank branches by just visiting the site [My Nearest Bank.](https://mynearestbank.com/)

**Reply by JohnTitor** [reply]
Note that GitHub Sponsors currently use a local currency, which is used on Stripe, for monthly receipts. That is, calculating the amount by year is _possible_ by summing them up. Actually, the pull-down menu on the web UI shows the amount in my local currency.

**Reply by kissu** [reply]
Dead link above.

@JohnTitor I'll give a look on Stripe thanks!

**Comment by vistonviston** [comment]
This question remains relevant year after year. If you want to learn about [tax season 2024](https://help-center.pissedconsumer.com/getting-ready-for-the-new-tax-season/), I recommend reading this article. It provides useful information about taxes and their payment in 2024. Remember that proper preparation will help you avoid tax mistakes and you will be confident and calm. Moreover, if you want to file your income taxes on time, you’ll need to file by the tax deadline.

**Comment by kissu** [comment]
This feature would be nice indeed. Might simply the life of any tax accountant.
